probe-rs/src/flashing/flasher.rs

```diff
@@ -244,44 +222,289 @@ impl<'a> Flasher<'a> {
         Ok(r)
     }
 
+    /// Writes a single block of data to a given address in the flash.
+    ///
+    /// This will not check any boundaries. You have to make sure that the boundaries are correct.
```

Comment:
What boundaries are meant here? Just the boundaries of the flash, or  page/sector boundaries?

Reply:
I adjusted the docstring do be more precise :)